File: 02_ProgrammingFundamentals/_10_TextProcessing/_02_Exercise/_02_CharacterMultiplier.java
package _02_ProgrammingFundamentals._10_TextProcessing._02_Exercise;

import java.util.Scanner;

public class _02_CharacterMultiplier {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        String[] console = scan.nextLine().split("\\s+");

        // Add variables
        String firstWord = console[0];
        String secondWord = console[1];
        int sum = 0;

        // Multiply characters
        if (firstWord.length() > secondWord.length()) {
            for (int i = 0; i < secondWord.length(); i++) {
                sum += firstWord.charAt(i) * secondWord.charAt(i);
            }

            for (int i = secondWord.length(); i < firstWord.length(); i++) {
                sum += firstWord.charAt(i);
            }
        } else if (firstWord.length() < secondWord.length()) {
            for (int i = 0; i < firstWord.length(); i++) {
                sum += firstWord.charAt(i) * secondWord.charAt(i);
            }

            for (int i = firstWord.length(); i < secondWord.length(); i++) {
                sum += secondWord.charAt(i);
            }
        } else {
            for (int i = 0; i < secondWord.length(); i++) {
                sum += firstWord.charAt(i) * secondWord.charAt(i);
            }
        }

        // Print result
        System.out.println(sum);

        scan.close();
    }
}

File: 24_C#Fundamentals/_08_TextProcessing/_02_Exercise/_02_CharacterMultiplier/_02_CharacterMultiplier.cs
using System;

public class _02_CharacterMultiplier
{
    public static void Main(string[] args)
    {
        // Read input values
        string[] console = Console.ReadLine().Split(" ");

        // Add variables
        string firstWord = console[0];
        string secondWord = console[1];
        int sum = 0;

        // Multiply characters
        if (firstWord.Length > secondWord.Length)
        {
            for (int i = 0; i < secondWord.Length; i++)
            {
                sum += firstWord[i] * secondWord[i];
            }

            for (int i = secondWord.Length; i < firstWord.Length; i++)
            {
                sum += firstWord[i];
            }
        }
        else if (firstWord.Length < secondWord.Length)
        {
            for (int i = 0; i < firstWord.Length; i++)
            {
                sum += firstWord[i] * secondWord[i];
            }

            for (int i = firstWord.Length; i < secondWord.Length; i++)
            {
                sum += secondWord[i];
            }
        }
        else
        {
            for (int i = 0; i < secondWord.Length; i++)
            {
                sum += firstWord[i] * secondWord[i];
            }
        }

        // Print result
        Console.WriteLine(sum);
    }
}
